plugins/modules/cloud/docker/docker_container.py

```diff
@@ -212,6 +212,40 @@
         - "Must be a positive integer."
         type: int
         required: yes
+  device_requests:
+    description:
+      - Allows to request additional resources, such as GPUs.
+    type: list
+    elements: dict
+    suboptions:
+      capabilities:
+        description:
+          - List of lists of strings to request capabilities.
+          - The top-level list entries are combined by OR, and for every list entry,
+            the entries in the list it contains are combined by AND.
+          - The driver will try to satisfy one of the sub-lists.
```

Comment:
```suggestion
          - The driver will try to satisfy one of the sub-lists.
```
would be good to rephrase this to use a present tense as recommended in best practices.

Reply:
Done in 36b2645db46ecc19adde875b1f2d85dd2b1995a9.